data/core/init.lua

```diff
@@ -691,14 +696,16 @@ function core.init()
   cur_node = cur_node:split("down", core.status_view, {y = true})
 
   command.add_defaults()
-  local got_user_error = not core.load_user_directory()
-  local plugins_success, plugins_refuse_list = core.load_plugins()
-
+  local got_user_error = not core.load_user_directory()  
   do
     local pdir, pname = project_dir_abs:match("(.*)[/\\\\](.*)")
     core.log("Opening project %q from directory %s", pname, pdir)
   end
   local got_project_error = not core.load_project_module()
+  for i = 1, #core.project_directories do
+    scan_project_folder(i)
+  end
+  local plugins_success, plugins_refuse_list = core.load_plugins()
```

Comment:
Wouldn't it be better to load the plugins before scanning the project? This way if for example a plugin overrides `config.ignore_files` we would scan the correct files/dirs.

Reply:
I'd say no; ideally project would be loaded before, because it may want to configure plugins (same as with the user plugin; disabling certain plugins, configuring others on load). I honestly consider the project plugin to be a logical extension of hte user plugin. And, plus, not all plugins are susceptible to config changes after load.

Reply:
Oh, I see what you're saying. In that case, yes. Actually, you're right. But only if we don't have to scan the folder to read the project file (we shouldn't); but I think I had some trouble doing that. But it probably will just take a minor refactoring.

So yeah, I'll try and do that.

Reply:
Oh yeah; sorry. No refactoring even necessary. Good call.